src/features/content-answers/ContentAnswersModalFooterActions.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/variables';
+
+.bdl-ContentAnswersModalFooterActions {
+    position: relative;
+    top: -$bdl-grid-unit * 4;
+    right: 0;
+    left: 0;
+    align-self: flex-start;
+    width: 100%;
+    padding: $bdl-grid-unit * 3 0 0 0;
```

Comment:
some of the styles here are a bit curious. it's adds a top padding but then re-positions it?

my impression is that it just needs a negative `margin-top` of 4px? the parent `bdl-ContentAnswersModalFooter` is an unstyled div so i'm thinking it doesn't need the other stuff

Reply:
![image](https://github.com/box/box-ui-elements/assets/103291617/c8dc07f7-a1d2-48c2-a273-31fe8dde77c8)
without padding but top 
![Screenshot 2023-10-24 at 4 19 35 PM](https://github.com/box/box-ui-elements/assets/103291617/6f562122-5593-4a35-9f68-7d0dc89319e7)
with padding but top
![image](https://github.com/box/box-ui-elements/assets/103291617/8b3c67bb-21e4-465e-956b-4f99ae368459)
no padding or top 
![image](https://github.com/box/box-ui-elements/assets/103291617/bb66a33e-31f1-4b8b-b84b-a5c034799125)
no top but padding
![image](https://github.com/box/box-ui-elements/assets/103291617/9f772ccc-ce52-4301-bb4a-a9fad3b4d2f5)

with both

Reply:
![image](https://github.com/box/box-ui-elements/assets/103291617/fcb7c2b8-f6e5-401c-be4d-530efe5abc71)
no padding or top but margin-top -4 px

Reply:
did the margin-top solution work?

my other thinking was that `position: relative; align-self: flex-start; width: 100%;` were unnecessary as well.